second class/second.js

```diff
@@ -0,0 +1,53 @@
+console.log("Hello for js 2");
+
```

Comment:
Always try to put a space before and after a equal sign. That is a good practice indeed.